entries/z/zip.co.json

```diff
@@ -0,0 +1,20 @@
+{
+	"Zip": {
+		"domain": "zip.co",
+		"tfa": [
+			"sms"
+		],
+		"documentation": "https://zip.co/security",
```

Comment:
The documentation page was great to verify that the service indeed supports 2FA but I don't think it will tell users anything of value. I recommend you remove the documentation link.
```suggestion
```

Reply:
I did think it was a little pointless to add to considering you don't have to do anything to enable it. But wasn't too sure